### ISSUE TITLE
coursier-cli_2.11-1.0.0-M15-standalone.jar: Connection timed out

### ISSUE THREAD

**Issue by Habulinec** [alexarchambault/coursier#425]
Hello,

I have an issue when running the ./coursier --help command: 

```
./coursier --help
Error while downloading https://oss.sonatype.org/content/repositories/releases/io/get-coursier/coursier-cli_2.11/1.0.0-M15/coursier-cli_2.11-1.0.0-M15-standalone.jar: Connection timed out (Connection timed out), ignoring it
```

Which is strange, because when I curl the https link above, I can download the file normally.

(Red Hat Enterprise Linux Server release 7.2 (Maipo))
java -version
openjdk version "1.8.0_111"
OpenJDK Runtime Environment (build 1.8.0_111-b15)
OpenJDK 64-Bit Server VM (build 25.111-b15, mixed mode)

Do you know, how to troubleshoot this error?

Thanks a lot for help or info in advance.

Best regards

- Habu

**Comment by philippw** [comment]
Try adding `-1-`
  https://oss.sonatype.org/content/repositories/releases/io/get-coursier/coursier-cli_2.11/1.0.0-M15-1/coursier-cli_2.11-1.0.0-M15-1-standalone.jar

**Comment by Habulinec** [comment]
Hi @philippw  , sorry for not being around for awhile. Also thank you for your response, but frankly I don't get it. How should I tell the coursier to use the url you've provided?

**Comment by philippw** [comment]
@Habulinec 
The version string in the README is wrong.
This:
`addSbtPlugin("io.get-coursier" % "sbt-coursier" % "1.0.0-M15")`

should actually be:
`addSbtPlugin("io.get-coursier" % "sbt-coursier" % "1.0.0-M15-1")`

**Comment by Habulinec** [comment]
Ah, never mind, it was a network issue. :) Apparently Coursier was trying to get the files through some port, which is most probably blocked by our company firewall. I've tried it home with a direct internet connection and it works without any modification.

Sorry for the confusion. ;)

**Comment by shengc** [comment]
sorry to comment on a closed issue: Behind a proxy too, I am wondering where the standalone jar is downloaded to ? I am thinking I can download that jar manually through a direct curl, and then put it in place so next time I run `coursier`, it won't try to download the same jar again.

**Comment by cequencer** [comment]
I am running into the same issue, since this issue is closed, I opened a new issue #888 to track it.  

@shengc 
The default cache directory for coursier-cli is your home directory's ~/.coursier
You can in fact launch coursier-cli on a computer without firewall and tar up ~/.coursier and move that tar-ball to the computer behind firewall to get over this blocker and get a jumpstart.

**Comment by cequencer** [comment]
Even with the hack to get coursier bootstrapped with the required set of artifacts, I am still blocked during runtime when more artifacts is being pulled during runtime.  I have made further ideas and comment in issue #888.

[I 23:14:27.708 NotebookApp] Kernel started: NOTEBOOK_UUID
[W 23:14:38.173 NotebookApp] Timeout waiting for kernel_info reply from NOTEBOOK_UUID
Downloading https://repo1.maven.org/maven2/org/scala-lang/modules/scala-xml_2.11/1.0.5/scala-xml_2.11-1.0.5.pom.sha1
Downloading https://repo1.maven.org/maven2/org/scala-lang/modules/scala-xml_2.11/1.0.5/scala-xml_2.11-1.0.5.pom
